Corona-Warn-App/src/main/java/de/rki/coronawarnapp/bugreporting/ui/ErrorDialog.kt

```diff
@@ -60,3 +60,14 @@ fun Throwable.toErrorDialogBuilder(context: Context) =
             }.show()
         }
     }
+
+fun Throwable.toErrorDialogBuilderNoDetails(context: Context) =
+    MaterialAlertDialogBuilder(context).apply {
+        val error = this@toErrorDialogBuilderNoDetails
+        val humanReadable = error.tryHumanReadableError(context)
+
+        setTitle(humanReadable.title ?: context.getString(R.string.errors_generic_headline_short))
+        setMessageView(humanReadable.description, textHasLinks = true)
+
+        setPositiveButton(R.string.errors_generic_button_positive) { _, _ -> }
+    }
```

Comment:
Does this require a new extension function?

If we use `toErrorDialogBuilder` and then call `setNeutralButton` on the builder, we overwrite the previous neutral?